.github/workflows/ray-docker.yml

```diff
@@ -0,0 +1,51 @@
+name: ray-docker
+on: pull_request
+jobs:
+  test-ray-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - shell: bash -l {0}
+        run: bash examples/docker/ci-ray-docker/build-docker-image.sh
```

Comment:
I wonder if using something like this https://github.com/marketplace/actions/docker-run-action#run-an-image-built-by-a-previous-step would not look nicer... I do not like a lot of copy-pasting of `docker exec ray_container pytest` thing

Reply:
Yes, this approach allows to avoid code duplication, so action was used.